Makefile

```diff
@@ -218,6 +218,7 @@ deps: $(PLUGIN_DIR) $(GO_ETHEREUM_DIR) $(SSHA3_DIR)
 		github.com/prometheus/client_golang/prometheus \
 		github.com/go-kit/kit/log \
 		github.com/BurntSushi/toml \
+		github.com/tidwall/gjson \
```

Comment:
This should go into `Gopkg.toml`, we only `go get` packages shared with `go-loom`.
You can use `dep ensure -add github.com/tidwall/gjson` to update `Gopkg.toml` & `Gopkg.lock.